src/main/java/net/william278/profanitycheckerapi/ProfanityChecker.java

```diff
@@ -66,6 +70,90 @@ public double getTextProfanityLikelihood(String text) {
         return interpreter.getValue("predict_prob([text])[0].item()", Double.class);
     }
 
+    /**
+     * Determines whether a string of text contains profanity
+     *
+     * @param text The string of text to check
+     * @return {@code true} if the text likely contains profanity; {@code false} otherwise
+     */
+    public boolean isTextProfaneBypass(String text) {
+        Set<String> toCheck = getTextToCheck(text);
+        for (String s : toCheck) {
+            interpreter.set("text", s);
+            boolean result = interpreter.getValue("predict([text])[0].item()", Integer.class) == 1;
```

Comment:
Instead of manually doing this, let's reduce the repetition here by just calling our existing `#isTextProfane(text)` method